docs/publications/posts/2024/2024-boring-problems-tn.md

```diff
@@ -0,0 +1,25 @@
+---
+title: "A Chat about Boring Problems: Studying GPT-Based Text Normalization"
+author: [Yang Zhang, Travis M. Bartley, Mariana Graterol-Fuenmayor, Vitaly Lavrukhin, Evelina Bakhturina, Boris Ginsburg]
+author_gh_user: []
+readtime: 30
+date: 2024-04-14
+
+# Optional: Categories
+categories: [Text Normalization, Large-Language Models]
+continue_url:https://ieeexplore.ieee.org/abstract/document/10447169
```

Comment:
please add a space after the colon, otherwise the docs won't build

Reply:
fixed